packages/core/src/html/MdAttributeRenderer.js

```diff
@@ -105,8 +105,12 @@ class MdAttributeRenderer {
    */
 
   processPanelAttributes(node) {
-    this.processAttributeWithoutOverride(node, 'alt', false, '_alt');
-    this.processAttributeWithoutOverride(node, 'header', false);
+    if (!this.hasSlotOverridingAttribute(node, 'alt')) {
+      this.processAttributeWithoutOverride(node, 'alt', false, '_alt');
+    }
```

Comment:
We are not currently supporting the `alt` attribute as a slot. I believe the `processAttributeWithoutOverride` for `alt` is because `alt` clashes with an existing HTML attribute, so we want to convert it to `_alt` instead. So we won't need to throw a warning for this :)

Reply:
Noted! I have removed the warning in [25f4250](https://github.com/MarkBind/markbind/pull/2053/commits/25f42501e8da056540cb26e7b51f23ce0746c88f) :+1: